verilog/analysis/checkers/signal_name_style_rule.cc

```diff
@@ -43,17 +48,37 @@ using verible::LintViolation;
 using verible::SyntaxTreeContext;
 using verible::matcher::Matcher;
 
-static constexpr absl::string_view kMessage =
-    "Signal names must use lower_snake_case naming convention.";
+#define STYLE_DEFAULT_REGEX "[a-z_0-9]+"
+static std::string style_default_regex = STYLE_DEFAULT_REGEX;
+
+SignalNameStyleRule::SignalNameStyleRule() {
+  style_regex_ =
```

Comment:
use RAII instead of assigning values inside the constructor.

So
```c++
SignalNameStyleRule::SignalNameStyleRule() 
  : style_regex_(new re::RE2(...)), violation_message_(absl::StrCat("...")) {}
```

Reply:
No problem. Done.